src/RegisterService/Controllers/ApiController.cs

```diff
@@ -0,0 +1,14 @@
+using Microsoft.AspNetCore.Mvc;
+
+namespace ProdtestApi.Controllers;
+
+[ApiController]
+[Route("[controller]")]
+public class ApiController : ControllerBase
+{
+    [HttpGet("health")]
+    public IActionResult HealthCheck()
+    {
+        return Ok("{\"health\": \"ok\" }");
```

Comment:
volgens mij heb je ook standaard middlewaren van .net die dit voor je regelt. Daar zit ook standaard een ok response aan vast. Eigenlijk wil je hier testen of alles wat je gebruikt in de service (connectie naar een service bus, connectie met de database) allemaal goed werkt.

Reply:
Yess, alleen heb ik dat nu nog niet. Voor nu wilde ik deze houden omdat 1) er in de toekomst wel connecties worden getest en 2) dit makkelijk laat zien dat de API draait.